.gitignore

```diff
@@ -9,3 +9,7 @@ buildNumber.properties
 .mvn/timing.properties
 # https://github.com/takari/maven-wrapper#usage-without-binary-jar
 .mvn/wrapper/maven-wrapper.jar
+item-handler.jar
+.DS_Store
+.idea/
+item-handler.iml
```

Comment:
Actually, more generic exclusions like we see in the other projects will reduce the risk of adding Idea project files.

Reply:
Good point. Done.